src/Function/Structures.agda

```diff
@@ -123,7 +139,16 @@ record IsRightInverse (to : A → B) (from : B → A) : Set (a ⊔ b ⊔ ℓ₁
     renaming (cong to to-cong)
 
   strictlyInverseʳ : StrictlyInverseʳ _≈₁_ to from
-  strictlyInverseʳ x = inverseʳ Eq₂.refl
+  strictlyInverseʳ = inverseʳ⇒strictlyInverseʳ _≈₁_ _≈₂_ Eq₂.refl inverseʳ
+
+  injective : Injective _≈₁_ _≈₂_ to
+  injective = inverseʳ⇒injective {f⁻¹ = from} _≈₂_ to Eq₂.refl Eq₁.sym Eq₁.trans inverseʳ
```

Comment:
Yuk! Can this not be refactored/improved in some way?
It certainly seems to hint that the parameterisation in `Function.Consequences` may be suboptimal?

Reply:
Most recent commit at least managed to remove `{f⁻¹ = from}`...

Reply:
Probably the best solution is to re-instantiate `module Section` in `Function.Consequences.Setoid` so that all the equality axioms are plumbed-in in place, and then import *that* in `Function.Structures` and `Function.Bundles`... d'oh!